include/ignition/physics/GetEntities.hh

```diff
@@ -51,6 +53,9 @@ namespace ignition
       };
     };
 
+    /////////////////////////////////////////////////
+    /// \brief This feature retrieves the index-specified World pointer from the
```

Comment:
this feature has API's for getting a world pointer by index or by name, but this feature-level description only says "index-specified"